protocols/earthfund/overview.md

```diff
@@ -0,0 +1 @@
+Overview of protocol and governance process.
```

Comment:
Please add a short description about Earthfund. 

## Earthfund
 - what is earthfund and what's your mission
 
## Overview 
- How are you working towards your mission? 
- Ex: Protocol overview , DAO structure overview

## Governance 
- How is your governance structured ?
- Is it on-chain or off-chain or both ?
- A short blurb about you gov token
- How are proposals created and reviewed ?

## Additional links
- Links to your social media profiles 
- Links to your gov-platform